sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -237,14 +237,6 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       intercept[AnalysisException] {
         sql(s"""LOAD DATA LOCAL INPATH "$incorrectUri" INTO TABLE non_part_table""")
       }
-
-      // Unset default URI Scheme and Authority: throw exception
```

Comment:
Because `fs.default.name` is deprecated for YARN implementations, but will still work. `fs.defaultFS` is the newer setting. `fs.default.name` is just an alias for it.

In `newHadoopConf`, the foreach loop of `conf.getAllConfs` will add `fs.defaultFS` setting value. So when we query `fs.default.name`, we still get the value.